test/e2e/spinup_test.go

```diff
@@ -100,28 +88,19 @@ func scraper(i int, config string, gossip bool) (cmdScheduleFunc, string) {
 			"--http-address", sidecarHTTP(i),
 			"--prometheus.url", fmt.Sprintf("http://%s", promHTTP(i)),
 			"--tsdb.path", promDir,
-			"--cluster.address", sidecarCluster(i),
+			"--cluster.disable",
 			"--log.level", "debug",
 		}
 
-		if gossip {
-			args = append(args, []string{
-				"--cluster.advertise-address", sidecarCluster(i),
-				"--cluster.gossip-interval", "200ms",
-				"--cluster.pushpull-interval", "200ms",
-			}...)
-			args = append(args, clusterPeerFlags...)
-		} else {
-			args = append(args, "--cluster.disable")
-		}
+		args = append(args, "--cluster.disable")
```

Comment:
we don't need this and test fail because we have repeating `--cluster.disable`

Reply:
and that what actually happen in CI..